### ISSUE TITLE
Clicking on Content's icon doesn't redirect

### ISSUE THREAD

**Issue by Mihaiii** [StarcounterApps/Launchpad#25]
### Steps to reproduce
1. https://github.com/StarcounterApps/Content/commit/018952ab38ea36ae4880058ba491b4d67007f900 (current latest version on develop)
2. Run https://github.com/StarcounterApps/Launchpad/commit/904fea359902a740a25b508d791b84ee06c38c9b (current latest version on develop)
3. Click on the Content app icon.
### Expected result
Redirects to `/content` (that redirects to `content/cms`)
### Actual result
Url changes to `/content`, but the page doesn't change. There's no error in console.
##### Starcounter version 
SC 2.3.1.7166
##### Browsers affected

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari

**Comment by Mihaiii** [comment]
The same things happen for "CompositionEditor" for example because in that case also the main pages redirect to another.

@warpech I'm not sure who I should tag here.

**Comment by Mihaiii** [comment]
This issue is not really related to Launchpad, but here it was noticed and I don't know who I should tag.
I ran Content app on `master` branch with current latest SC Release (2.3.0.6350) and it seems ok.

I'll try to isolate the SC version in this weekend if it's not solved by that time.

**Comment by Mackiovello** [comment]
So if I understand it correctly, it works if you refresh after the redirect? 

Has this worked on an earlier version? I remember struggling with redirects some time too.

**Comment by Mihaiii** [comment]
> So if I understand it correctly, it works if you refresh after the redirect?

Yes, since the url in the address bar is already `/content`. Refreshing on that works correctly (I might be wrong, I speak from memory). In other words, If the user accesses `/content` directly, everything is ok, but if I get to `/content` using an anchor/puppet-redirect, I have the above problem.

> Has this worked on an earlier version? I remember struggling with redirects some time too.

It works on `master` branch and with the latest Release in a sense that the URL changes to `/content` and the page changes too (again, I'm not very sure, I will test it).

If this is an unknown issue, please let me know and I'll isolate the SC version.
